android/app/src/main/java/org/fossasia/openevent/adapters/DayScheduleAdapter.java

```diff
@@ -107,6 +109,17 @@ public void onBindViewHolder(DayScheduleViewHolder holder, int position) {
         }
         holder.slotLocation.setText(currentSession.getMicrolocation().getName());
 
+        Track currentSessionTrack = currentSession.getTrack();
+        holder.slotTrack.setText(currentSessionTrack.getName());
```

Comment:
@shreyasnbhat Please look at this, this looks much cleaner and the font is thin

![](https://cloud.githubusercontent.com/assets/17262180/25820152/f398d3ee-344d-11e7-8805-70b49e467336.png)